server/command.go

```diff
@@ -183,23 +187,47 @@ func (p *Plugin) runDisconnectCommand(user *model.User) (string, error) {
 func (p *Plugin) runHelpCommand(user *model.User) (string, error) {
 	text := "###### Mattermost Zoom Plugin - Slash Command Help\n"
 	text += strings.ReplaceAll(helpText, "|", "`")
+	text += "\n" + strings.ReplaceAll(statusHelpText, "|", "`")
 
 	if p.canConnect(user) {
 		text += "\n" + strings.ReplaceAll(oAuthHelpText, "|", "`")
 	}
 	return text, nil
 }
 
+func (p *Plugin) runStatusChange(userID string, args []string) (string, error) {
+	errMsg := "Expecting yes/no"
+	if len(args) < 2 {
+		return errMsg, errors.New("unexpected argument")
+	}
+
+	action := args[1]
+	if action != yes && action != no {
+		return errMsg, errors.New("unexpected argument")
+	}
+
+	appErr := p.API.KVSet(fmt.Sprintf("%v_%v", changeStatusKey, userID), []byte(action))
+	if appErr != nil {
+		p.API.LogDebug("Could not save status change preference ", appErr)
+		return "Could not save status preference", appErr
+	}
+	return "Status preference updated successfully", nil
```

Comment:
Can this message convey the fact of "Your status will be updated" or "Your status will not be updated"?